crates/autopilot/src/decoded_settlement.rs

```diff
@@ -365,12 +362,30 @@ impl DecodedSettlement {
 pub struct Fees {
     /// The UID of the order associated with these fees.
     pub order: OrderUid,
+    /// The type of fee that was executed.
+    pub kind: FeeKind,
```

Comment:
If I understand correctly, the only reason for why we need this differentiation here is that the order executions table only stores surplus_fees (and not the fixed fee amounts). How hard would it be to change that and store total fee amount in that table?

Reply:
In order to store both surplus fee and user fee in order executions, we would have to:
1. Adjust executed_surplus_fee calculation in ORDERS db query
2. Probably some more adjustments in the solver rewards script to check for order type.

IMO this is wasteful effort because once we move to limit orders only, it's much more easier to just erase the `pub kind: FeeKind` line and dependent code.